pkg/acceptance/testing.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package acceptance
+
+import "context"
+
+// TestingT is the minimal subset of testing.TB needed for acceptance
+// tests.
+type TestingT interface {
+	Context() context.Context
+	TempDir() string
+	Helper()
+	Cleanup(func())
+	Log(args ...any)
+	Logf(format string, args ...any)
+	Error(args ...any)
+	Errorf(format string, args ...any)
+	Fatal(args ...any)
+	Fatalf(format string, args ...any)
+}
```

Comment:
Currently unused. I started with this but I think every testing framework should now support `testing.TB`.